sdk/webpubsub/azure-webpubsub-client/azure/webpubsub/client/_model_base.py

```diff
@@ -0,0 +1,706 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=protected-access, arguments-differ, signature-differs, broad-except
+# pyright: reportGeneralTypeIssues=false
+
+import functools
+import sys
+import logging
+import base64
+import re
+import copy
+import typing
+from datetime import datetime, date, time, timedelta, timezone
+from json import JSONEncoder
+import isodate
+from azure.core.exceptions import DeserializationError
+from azure.core import CaseInsensitiveEnumMeta
+from azure.core.pipeline import PipelineResponse
+from azure.core.serialization import NULL as AzureCoreNull
+
+if sys.version_info >= (3, 9):
+    from collections.abc import MutableMapping
+else:
+    from typing import MutableMapping
+
+_LOGGER = logging.getLogger(__name__)
+
+__all__ = ["NULL", "AzureJSONEncoder", "Model", "rest_field", "rest_discriminator"]
+
+
+class _Null(object):
+    """To create a Falsy object"""
+
+    def __bool__(self):
+        return False
+
+    __nonzero__ = __bool__  # Python2 compatibility
```

Comment:
shouldnt need this anymore as this is py3+

Reply:
same with https://github.com/Azure/azure-sdk-for-python/pull/28156#discussion_r1118233645